### PR TITLE
[READY] [SPEEDMERGE] She Comes.

### PR DESCRIPTION
**Pull request by CinderWC**
## About The Pull Request
This adds a new hairstyle into the game.

## Why It's Good For The Game
She hates the hairstyling options that are currently available

## Changelog
:cl:
add: Added modular code for a new hairstyle
imageadd: Added sprites for said hairstyle
/:cl:

### CODE REVIEW

File: modular_skyrat/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ///////////////////////////
-//Nice haircut liberal   //
+//  Modular Hairstyles   //
 ///////////////////////////
-/datum/sprite_accessory/hair/twintail_ass
+/datum/sprite_accessory/hair/twintail_ass // nice hairstyle, liberal
```
Comment (Trilbyspaceclone) [inline modular_skyrat/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:4]: image

Comment (CinderWC) [reply modular_skyrat/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:4]: PepeHands

File: modular_skyrat/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
   name = "Long twintails"
   icon = 'modular_skyrat/icons/mob/human_face.dmi'
   icon_state = "hair_twintail_ass"
+
+/datum/sprite_accessory/hair/viper // ty trilby - cinderwc
```
Comment (Trilbyspaceclone) [inline modular_skyrat/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:9]: image

Comment (CinderWC) [reply modular_skyrat/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:9]: PepeHands

**Pull request merged**